README.adoc

```diff
@@ -21,7 +21,7 @@ Suppose you want to create a Policy that control if request contains the ``X-Foo
 mvn archetype:generate\
  -DarchetypeGroupId=io.gravitee.maven.archetypes\
  -DarchetypeArtifactId=gravitee-policy-maven-archetype\
- -DarchetypeVersion=1.9.0\
+ -DarchetypeVersion=1.9.0-SNAPSHOT\
```

Comment:
Can you revert this change? The version will be 1.9.0 once published.

Reply:
Just to confirm - You want the version in the documentation to remain `1.9.0`, but pom to stay snapshot?